Dashboard/src/EntityDetails/EditEntity.js

```diff
@@ -41,6 +48,42 @@ const EditEntity = (props) => {
     })
   )
 
+  // Saved the modified data to an savedData and keep record of the indexes of table data cell
+  const saveData = (index, newData, action) => {
```

Comment:
1. Update comments for all methods please per conversation and standards.
2. If you are giving an action for this and remove data, make one function. If you take away the action, then you can keep the two functions

Reply:
Preferable just use one method